src/components/SiteListEntry/index.jsx

```diff
@@ -0,0 +1,33 @@
+import { useNavigate } from "react-router-dom";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+function SiteListEntry({ name, originUrl, logoUrl }) {
+  const navigate = useNavigate();
+
+  const handleSubmit = e => {
+    e.preventDefault();
+
+    navigate(`/genie-mode/${originUrl}`);
+  };
+
+  return (
+    <EntryWrapper onClick={handleSubmit}>
+      <img src={logoUrl} alt="site-logo"></img>
+      <p>{name}</p>
+      <p>{originUrl}</p>
+    </EntryWrapper>
+  );
+}
+
+SiteListEntry.propTypes = {
+  name: PropTypes.string.isRequired,
+  originUrl: PropTypes.string.isRequired,
+  logoUrl: PropTypes.string.isRequired,
+};
+
+export default SiteListEntry;
+
+const EntryWrapper = styled.div`
+  margin: 10px;
+`;
```

Comment:
```suggestion
const EntryWrapper = styled.div`
  margin: 10px;
`;

export default SiteListEntry;
```
p3 위와 같이 export 구문이 제일 아래에 있는 게 어떨까요?